spec/sideloading_spec.rb

```diff
@@ -337,10 +337,29 @@ def self.name
   describe 'polymorphic_belongs_to macro' do
     let!(:visa) { PORO::Visa.create }
     let!(:mastercard) { PORO::Mastercard.create }
+    let!(:paypal) { PORO::Paypal.create }
     let!(:employee2) do
       PORO::Employee.create credit_card_type: 'Mastercard',
           credit_card_id: mastercard.id
     end
+    let!(:employee3) do
+      PORO::Employee.create credit_card_type: 'Paypal',
+                            credit_card_id: paypal.id
+    end
+    let(:assert_correct_response) do
```

Comment:
Typically use `let` for setup and `def assert_correct_response` for assertions

Reply:
fixed.  Couldn't remember if rspec def defined outside a context and redefined inside a nested context would be fragile depending on execution order.